webapp/src/css/content-list.less

```diff
@@ -250,8 +244,110 @@
     &.selected {
       background-color: @tasks-highlight;
     }
-    > a:hover {
+    &:hover {
       border-left-color: @tasks-color;
     }
   }
 }
+
+.select-mode-available {
+  .content-row input[type="checkbox"],
+  .select-all input[type="checkbox"] {
+    accent-color: @blue-dark;
```

Comment:
Where is the existent palette? 
I thought anything from variable.less file was okay

Reply:
btw the blue is the same as in the Figma design

Reply:
So, at the top of `variables.less` there's [a comment](https://github.com/medic/cht-core/blob/master/webapp/src/css/variables.less#L1) 
```Basic colors. WARNING: Don't use these outside this file as the variable names are not useful````

The pallete is defined in `https://github.com/medic/cht-core/blob/master/webapp/src/css/variables.less#L26`

Reply:
Oh I completely didn't see that comment. I understand the reason, but in my experience that way has been unpractical. However I'll follow this way --- I'll add a `@checkbox-selected-color` since none of the existing ones describes properly the element.

Reply:
Thanks!

Reply:
[Ticket](https://github.com/medic/cht-core/issues/7862)